eli5/formatters/text.py

```diff
@@ -15,12 +17,15 @@
 _SPACE = '_' if six.PY2 else '░'
 
 
-def format_as_text(expl, show=fields.ALL, highlight_spaces=None):
+def format_as_text(expl, show=fields.ALL, highlight_spaces=None,
+                   show_feature_values=True):
     """ Format explanation as text.
     If ``highlight_spaces`` is None (default), spaces will be highlighted in
     feature names only if there are any spaces at the start or at the end of the
     feature. Setting it to True forces space highlighting, and setting it to False
     turns it off.
+    If ``show_feature_values`` is True (default), feature values are shown
+    if present.
```

Comment:
Ah, yet another gotcha - it should also be in [FORMAT_KWARGS](https://github.com/TeamHG-Memex/eli5/blob/master/eli5/ipython.py#L10).